components/navbar/StatsDropdown.vue

```diff
@@ -1,83 +1,128 @@
 <template>
   <div>
-    <b-dropdown
-      v-if="chain === 'bsx' || chain === 'snek'"
-      aria-role="list"
-      data-cy="stats"
-      :triggers="['click']">
-      <template #trigger>
-        <div class="navbar-item" data-cy="stats">
-          {{ $t('stats') }}
-        </div>
-      </template>
-      <b-dropdown-item has-link>
-        <nuxt-link
-          data-cy="global-offers"
-          :to="`${
-            accountId
-              ? `/${urlPrefix}/offers?target=${accountId}`
-              : `/${urlPrefix}/offers`
-          }`">
-          {{ $t('navbar.globalOffers') }}
-        </nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="offers-stats" :to="`/${urlPrefix}/stats`">
-          {{ $t('navbar.offerStats') }}
-        </nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="series-insight" to="/series-insight">
-          Series</nuxt-link
-        >
-      </b-dropdown-item>
-    </b-dropdown>
-    <b-dropdown v-if="chain === 'rmrk'" data-cy="stats" :triggers="['click']">
-      <template #trigger>
-        <div class="navbar-item" data-cy="stats">
-          {{ $t('stats') }}
-        </div>
+    <div v-if="!isMobile">
```

Comment:
Why this component has another +40% LOC?

Reply:
40% LOC? I dont understand ?

Reply:
LOC = Lines of code

The component magically grew to 120 Lines

Reply:
yeah because we have a different UI for mobile version so I added that in this component